env/src/test/java/com/sap/cloud/security/config/json/DefaultJsonObjectTest.java

```diff
@@ -3,8 +3,11 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package com.sap.cloud.security.json;
+package com.sap.cloud.security.config.json;
```

Comment:
package move by mistake?
In master config and json are on the same level.

Reply:
seems to be merging issue